### PR TITLE
Fix project links in work packages table

### PR DESCRIPTION
**Pull request by myabc**
https://www.openproject.org/work_packages/15518

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://fix-15518-links-to-project-in-project-column.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: app/assets/javascripts/angular/helpers/components/work-packages-helper.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ angular.module('openproject.workPackages.helpers')
           id = object.parent_id;
           break;
         case 'project':
-          id = object.project_id;
+          id = object.project.id;
```
Comment (ulferts) [inline app/assets/javascripts/angular/helpers/components/work-packages-helper.js]: @myabc works as desired. From a user perspective I would rather we take the identifier attribute as openproject usually hides the actual id of the project.

Comment (myabc) [inline app/assets/javascripts/angular/helpers/components/work-packages-helper.js]: @ulferts good point. I've updated this.

**Comment by ulferts** [comment]
:+1: thanks for the improvement

**Pull request merged**